harfbuzz-sys/makefile.touch

```diff
@@ -1,45 +0,0 @@
-# Check that $(HARFBUZZ) is defined before trying to use it.
```

Comment:
Is the removal of this file related to the rest of the change or could it be a separate PR?

Reply:
It is only used by that code in the build job that is building harfbuzz on Linux, so once that code disappeared in a commit, this went with it.